src/Compilers/CSharp/Portable/Binder/Binder_InterpolatedString.cs

```diff
@@ -54,6 +58,7 @@ private BoundExpression BindInterpolatedString(InterpolatedStringExpressionSynta
                             {
                                 alignment = GenerateConversionForAssignment(intType, BindValue(interpolation.AlignmentClause.Value, diagnostics, Binder.BindValueKind.RValue), diagnostics);
                                 var alignmentConstant = alignment.ConstantValue;
+                                constantEnabled = false;
                                 if (alignmentConstant != null && !alignmentConstant.IsBad)
```

Comment:
One potential benefit of going the `string.Format` route is that it could allow us to consider supporting alignment as that would come for free. We should keep that in mind when discussing that aspect of the implementation.

Reply:
Yup, talked it over with Fred. The alignment support for strings that can only be composed of constants may be outweighed by this implementation's default support for ropes.

---
In reply to: [447264298](https://github.com/dotnet/roslyn/pull/45396#discussion_r447264298) [](ancestors = 447264298)